INSTALL.md

```diff
@@ -388,8 +388,27 @@ for OpenSSL development.  It only works when using gcc or clang as the compiler.
 If you are developing a patch for OpenSSL then it is recommended that you use
 this option where possible.
 
-ZLib Flags
-----------
+Compression Algorithm Flags
+---------------------------
+
+### with-brotli-include
+
+    --with-brotli-include=DIR
+
+The directory for the location of the brotli include file.  This option is only
+necessary if [brotli](#brotli) is used and the include file is not
+already on the system include path.
+
+### with-brotli-lib
+
+    --with-brotli-lib=LIB
+
+**On Unix**: this is the directory containing the brotli library.
+If not provided the system library path will be used.
+
+**On Windows:** no support.
```

Comment:
Why no support on Windows?

Reply:
I'm not a Windows developer. I would accept any help getting brotli supported on Windows.
Note: this is a brotli limitation, not a certificate compression limitation.

Reply:
Windows support is now there; Windows CI has been updated to build/test with zstd and brotli.